.github/ISSUE_TEMPLATE/1.bug_report.yml

```
@@ -48,7 +48,7 @@ body:
         - 'TypeScript'
   - type: input
     attributes:
-      label: Link to reproduction - Issues with a link to complete (but minimal) reproduction code will be addressed faster
+      label: Link to complete (but minimal) reproduction code - it will allow us to quickly check and debug this issue
```

```suggestion
      label: Link to the code that reproduces this issue
```

I think we can keep this label short. The description below explains more.

---

:+1: